app/lib/l10n/app_ur.arb

```diff
@@ -2144,7 +2144,7 @@
   "@audio": {},
   "pinDetails": "پن کی تفصیلات",
   "@pinDetails": {},
-  "inSpaceLabelInline": "میں",
+  "inSpaceLabelInline": ":میں",
   "@inSpaceLabelInline": {},
   "comingSoon": "ابھی تک تعاون یافتہ نہیں، جلد آرہا ہے!",
```

Comment:
Also, would be great if you can correct the exclamation here in `comingSoon`, it should be at the end (from RTL) rather than at the start :)

Reply:
you know you can just do it? either in weblate or within a PR? I am not confident I understood what change you want and I'd rather you fix it ...

Reply:
Sure, I can take care of that in weblate.